apps/juicebox_web/assets/js/reactions/video-frame.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component, PropTypes } from 'react';
+const frameAsset = require.context('../../images/frames', false);
```

Comment:
It's no more standards-compliant than this anyway. I think the only way to do it that would be would be to have a manifest file where you export the frames.